src/cli/utils/restore_config.go

```diff
@@ -23,10 +24,18 @@ type RestoreCfgOpts struct {
 	ProtectedResource string
 	SkipPermissions   bool
 
+	// SubService is useful when we are trying to restore Groups where
+	// the user can only restore a single service at any point instead of
+	// the entire backup.
+	SubServiceType path.ServiceType
+	SubService     string
```

Comment:
This seems to duplicate the selector's ownership.  Moving these out of there and into the restore config opens up the potential for conflict between these props and the selector.  Similarly, it might lead to building this data into the selector downstream anyway.  Iirc, the selector already has the capacity to specify the secondary resource; seems like this identification could be owned there instead.